src/python/pybind11/src/pyxrt.cpp

```diff
@@ -61,6 +62,19 @@ py::enum_<ert_cmd_state>(m, "ert_cmd_state")
     .value("ERT_CMD_STATE_SKCRASHED", ert_cmd_state::ERT_CMD_STATE_SKCRASHED)
     .value("ERT_CMD_STATE_MAX", ert_cmd_state::ERT_CMD_STATE_MAX);
 
+py::enum_<xrt::info::device>(m, "xrt_info_device")
+    .value("bdf", xrt::info::device::bdf)
+    .value("interface_uuid", xrt::info::device::interface_uuid)
+    .value("kdma", xrt::info::device::kdma)
+    .value("max_clock_frequency_mhz", xrt::info::device::max_clock_frequency_mhz)
+    .value("m2m", xrt::info::device::m2m)
+    .value("name", xrt::info::device::name)
+    .value("nodma", xrt::info::device::nodma)
+    .value("offline", xrt::info::device::offline)
+    .value("power_rails", xrt::info::device::power_rails)
+    .value("thermals", xrt::info::device::thermals)
+    .value("power_consumption", xrt::info::device::power_consumption)
+    .value("fans", xrt::info::device::fans);
```

Comment:
These have changed: https://github.com/Xilinx/XRT/pull/5522

Reply:
Rebased and updated code.